### PR TITLE
screen-headers main structure

### PR DESCRIPTION
**Pull request by ArtemLutsenko**

### CODE REVIEW

File: frontend/screen/src/components/Header/CurWeather/CurWeather.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.header-weather-info-item{
+  margin-left: 0;
+  font-family: SF UI Display;
+  font-size: 18px;
+  line-height: 16px;
+  letter-spacing: 0.00636364em;
+  color: #585858;
+}
+/*
+.header-current-weather{
```
Comment (javasia) [inline frontend/screen/src/components/Header/CurWeather/CurWeather.scss]: no comments are allowed in the master branch

File: frontend/screen/src/components/hooks/Menu/index.scss [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
+}
+.menu-hamburger{
     display: block;
-    float: left;
-    text-align: left;
+    margin-right: 102px;
 }
 
-.menu > ul{
+/*.menu > ul{
```
Comment (javasia) [inline frontend/screen/src/components/hooks/Menu/index.scss]: no comments

File: frontend/screen/src/components/hooks/Menu/index.scss [hunk 3]
```diff
@@ -1,33 +1,49 @@
...
 
-.menu li{
+/*.menu li{
     margin-bottom: 2em;
     color: black;
     list-style: none;
-}
+}*/
 
 a {
```
Comment (javasia) [inline frontend/screen/src/components/hooks/Menu/index.scss]: maybe we should use a more specific selector here as except the text-decoration prop your styles here are not likely to be used uniformly for all the links.

File: frontend/screen/src/components/Header/CurTime/CurTime.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      this.setState({
+        currentTime: new Date()
+      })
+    }, 1000)
+  }
+
+  render () {
+    return (
+      <div className='current-time'>
+        <div className={'current-time-main'}>{this.state.currentTime.toLocaleTimeString('en-GB', this.timeOptions)}</div>
```
Comment (zozich) [inline frontend/screen/src/components/Header/CurTime/CurTime.js]: 1. Content of long div's should be placed on a new line
2. Let's use BEM for class names. This should be `current-time__main`

File: frontend/screen/src/components/Header/CurTime/CurTime.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+.current-time-main{
+  margin-right: 0;
+  padding-right: 0;
+  color: #000000;
+  font-size: 44px;
+  line-height: 20px;
+  letter-spacing: .00636364em;
+  text-align: right;
+}
+.current-time-day{
```
Comment (zozich) [inline frontend/screen/src/components/Header/CurTime/CurTime.scss]: Let's always add one empty line between SCSS selectors

File: frontend/screen/src/components/Header/CurTrafficConditions/CurTrafficConditions.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react'
+import {ReactComponent as TrafCar} from '../../../img/icons/car.svg'
+import './CurTrafficConditions.scss'
+
+export default class CurTrafficConditions extends Component {
+  render () {
+    return (
+      <div className='header-traffic-condition'>
```
Comment (zozich) [inline frontend/screen/src/components/Header/CurTrafficConditions/CurTrafficConditions.js]: 1. BEM
2. Remove prefix `header`

File: frontend/screen/src/components/Header/CurTrafficConditions/CurTrafficConditions.scss [hunk 7]
```diff
@@ -0,0 +1,27 @@
+.header-traffic-condition{
+  margin: 0;
+  margin-right: 31px;
+  padding: 0;
+}
+
+.header-traffic-condition-icon {
```
Comment (zozich) [inline frontend/screen/src/components/Header/CurTrafficConditions/CurTrafficConditions.scss]: All subsequent selectors should be nested

File: frontend/screen/src/components/hooks/Cinema/index.js [hunk 8]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 const Cinema = (props) => (
```
Comment (zozich) [inline frontend/screen/src/components/hooks/Cinema/index.js]: Is it still used anywhere? If not, delete it

File: frontend/screen/src/components/AppRoutes/index.js [hunk 9]
```diff
@@ -1,29 +1,24 @@
...
-          <Route path="/FunnyCats" component={FunnyCats}/>
-          <Route path="/Extra" component={Extra}/>
-          <Route path="/" component={Restaurants}/>
-        </Switch>
-      </div>
-    )
+  return (
+    <div>
+      <Switch>
+        <Route path="/Services" component={Services}/>
```
Comment (zozich) [inline frontend/screen/src/components/AppRoutes/index.js]: All path items should be lowercase

File: frontend/screen/src/components/hooks/Menu/index.js [hunk 10]
```diff
@@ -1,21 +1,26 @@
 import React from 'react'
+import {ReactComponent as ServicesLogo} from '../../../img/icons/services.svg'
```
Comment (zozich) [inline frontend/screen/src/components/hooks/Menu/index.js]: Use import from root - `/img/icons/services.svg`

File: frontend/screen/src/components/Header/CurTime/CurTime.scss [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+  margin-top: 12px;
+  margin-right: 0;
+  padding-right: 3px;
+  color: #303030;
```
Comment (zozich) [inline frontend/screen/src/components/Header/CurTime/CurTime.scss]: Let's put all colors in a separate SCSS file `variables.scss` as variables. Create folder `styles` in root, and keep it there

**Pull request merged**